config/examples/Tronxy/X5SA/HOWTO-INSTALL.md

```diff
@@ -0,0 +1,280 @@
+# Table of Contents <!-- omit in toc -->
+- [Save Printer Settings (Optional, but recommended)](#save-your-printer-settings-optional-but-recommended)
+- [Configuring Marlin for your Printer](#configuring-marlin-for-your-printer)
+  - [1. Board Version](#1-board-version)
+  - [2. Bed size](#2-bed-size)
+  - [3. Stepper Drivers](#3-stepper-drivers)
+  - [4. Steps / mm](#4-steps--mm)
+  - [5. Other Marlin Config](#5-other-marlin-config)
+- [Backup your Chitu Firmware (Optional, but strongly recommended)](#backup-your-chitu-firmare-optional-but-strongly-recommended)
+- [Flashing Marlin JUST USING SD](#flashing-marlin-just-using-sd)
+- [Flashing Marlin Firmware MANUALLY (OBSOLETE!)](#flashing-marlin-firmware-manually-obsolete)
+- [Known Issues](#known-issues)
+
+# Save Printer Settings (Optional, but recommended)
+
+You can (OR MUST) dump the current settings of your printer. It may help you to figure out some configs, like steps/mm of your extruder.
+
+1. Create a file named `savesettings.gcode` with the following content:
+```
+M6046 ; sdcard access
+M8512 "currentconfig.gcode" ; save settings to file
+```
+2. Save it on the printer's SD card
+3. Put the card in the printer and "print" this file
+4. The printer won't do anything. Just wait a few seconds and stop the print.
+5. Your current printer settings are stored in the file: `currentconfig.gcode`
+
+You can read more about it in [this guide](https://www.facebook.com/notes/tronxy-turnigy-x5s-x5sa-x3s-3d-printer-drucker-users/tronxy-firmware-configuration-guide-by-keith-varin-addermk264bit-tuning/649799805579765/).
```

Comment:
This link is broken.

Reply:
The link works for group members. I just put it in the tutorial to properly credit the author.

Reply:
Could we have a link to something that is "open source"?

Reply:
Anyone can join the group (https://www.facebook.com/groups/169728600253557/about)

Reply:
The main propose here is just credit the author. This "save" methods was discovered by this user, and posted there.